assets/src/edit-story/components/button/index.js

```diff
@@ -126,3 +126,13 @@ export const GridView = ( props ) => (
 		<GridViewIcon />
 	</StyledButton>
 );
+
+export const ActionButton = styled.button.attrs( { type: 'button' } )`
```

Comment:
A `<button />` inside a `<form />` defaults to `type="submit"` so to avoid making the button submit the form for no reason, `type="button"` has to be added.

Reply:
TIL